sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala

```diff
@@ -290,6 +289,28 @@ private[ui] class ExecutionPagedTable(
         }
       }
 
+    var headerNameWithTooltips: Map[String, String] = Map(
+      "ID" -> "Execution ID of the SQL query.",
+      "Description" -> ("Description of the SQL query. When click on the description link," +
+        " it will navigate to query details page."),
+      "Submitted" -> "Submitted",
+      "Duration" -> "Difference between start time and close time.")
+
+      if (showRunningJobs && showSucceededJobs && showFailedJobs) {
+        headerNameWithTooltips += (
+          "Running Job IDs" -> "Id's of running jobs.",
+          "Succeeded Job IDs" -> "Id's of succeeded jobs.",
+          "Failed Job IDs" -> "Id's of failed jobs.")
+      } else if (showSucceededJobs && showFailedJobs) {
+        headerNameWithTooltips += (
+          "Succeeded Job IDs" -> "Id's of succeeded jobs.",
```

Comment:
Id's -> IDs
succeeded -> successful

Reply:
Thank you @srowen , i will modify tooltips as per your suggestion.